src/styles/block/catalog--card-stule.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  box-sizing: border-box;
+  height: 408px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    margin: 32px 0 36px;
+    text-align: center;
+  }
+
+  &__photo {
+    height: 134px;
+  }
+
+  &__info {
```

Comment:
you could just add padding inline to `.card` and get rid of this `card__info` wrapper